server/sonar-server/src/main/java/org/sonar/server/authentication/SsoAuthenticator.java

```diff
@@ -114,7 +119,10 @@ public void stop() {
     try {
       return doAuthenticate(request, response);
     } catch (BadRequestException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)